activiti-cloud-starter-configure/src/main/java/org/activiti/spring/boot/AbstractProcessEngineAutoConfiguration.java

```diff
@@ -118,6 +120,8 @@ protected SpringProcessEngineConfiguration baseSpringProcessEngineConfiguration(
       conf.setIdGenerator(new StrongUuidGenerator());
     }
     
+    ((ProcessEngineConfigurationImpl) conf).setActivityBehaviorFactory(new CloudActivityBehaviorFactory());
```

Comment:
@daisuke-yoshimoto can we avoid doing this? We don't have this for ServiceTasks, am I right? Let's just do it in the same way as service tasks.. by extending the engine.

Reply:
I do not want Activiti Cloud implementation in Activiti Engine, so I created a new ActivityBehaviorFactory for Activiti Cloud. If you are concerned about consistency, I can move MQServiceTaskBehavior.class processing to here.

Reply:
We don't want Activiti Cloud implementations inside the Engine, that's ok. But we manage to override the Bean selection inside the engine when we run in Cloud Mode (Runtime Bundle). We need to replicate this for the Signalling Behaviour